frontend/src/styles/App.css

```diff
@@ -129,3 +129,30 @@ textarea:focus-within {
 ::-webkit-scrollbar-thumb:hover {
   background-color: #4d94a8;
 }
+
```

Comment:
Use variable from pre-existed css variables from _variables.css
Rename class to `button--spinner` and all the button related css must be on _button.css
You can read more about style guides here:
https://github.com/divanov11/Mumble/blob/master/STYLE_GUIDE.md

Reply:
okay cool

Reply:
> Use variable from pre-existed css variables from `_variables.css`
> Rename class to `button--spinner` and all the button related css must be on `_button.css`
> You can read more about style guides here:
> https://github.com/divanov11/Mumble/blob/master/STYLE_GUIDE.md

@Mr-spShuvo I guess this solves everything. You can check it out.